packages/opentelemetry-semantic-conventions/src/trace/database.ts

```diff
@@ -19,12 +19,24 @@
  * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md
  */
 export const DatabaseAttribute = {
-  // db (required)
-  DB_TYPE: 'db.type',
+  // required
+  DB_SYSTEM: 'db.system',
   DB_INSTANCE: 'db.instance',
-  DB_STATEMENT: 'db.statement',
+  DB_STATEMENT: 'db.statement', // if applicable
+  DB_OPERATION: 'db.operation', // only required if db.statement is not applicable
```

Comment:
should these be tsdoc omments @dyladan ?

Reply:
in particular the second "only required if db.statement is not applicable"

Reply:
Moving to tsdoc makes sense to me. At some point, I think the tsdoc for each enum should be a copy/paste from the spec.

Reply:
I think it makes sense to move them to tsdoc yes

Reply:
> I think the tsdoc for each enum should be a copy/paste from the spec.

I'll try to include that in this PR!

Reply:
Added the tsdocs, please have a look!